mediagrains/cogenums.py

```diff
@@ -25,7 +25,17 @@
 
 from enum import IntEnum
 
-__all__ = ['CogFrameFormat', 'CogFrameLayout', 'CogAudioFormat', 'COG_FRAME_IS_PACKED', 'COG_FRAME_IS_COMPRESSED', 'COG_FRAME_FORMAT_BYTES_PER_VALUE']
+__all__ = [
+    'CogFrameFormat',
+    'CogFrameLayout',
+    'CogAudioFormat',
+    'COG_FRAME_IS_PACKED',
```

Comment:
Add a `COG_FRAME_IS_PLANAR` to simplify `not COG_FRAME_IS_PACKED(fmt) and not COG_FRAME_IS_COMPRESSED(fmt)`?